### PR TITLE
added event measurement script

### PR DESCRIPTION
**Pull request by stefan-lifebit**
**WHY**

- We need a script to evaluate our models on staging and prod.

**What?**

- Added a script for Event model evaluation

### CODE REVIEW

File: measurement_scripts/event_measurement.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                assert len(response["events"]) == tp + fp
+            except Exception:
+                raise MeasurementScriptBug("Bug in NER measurement script!")
+            true_positive += tp
+            false_negative += fn
+            false_positive += fp
+        except Exception as e:
+            logging.error(e)
+    logging.info(f"FP: {false_positive}, TP: {true_positive}, FN: {false_negative}")
+    return false_positive, true_positive, false_negative
```
Comment (ankushc48) [inline measurement_scripts/event_measurement.py:103]: Same here as in ner script.

File: measurement_scripts/tag_measurement.py [hunk 2]
```diff
@@ -89,9 +89,9 @@ def test_endpoint(url: str, data: List[Dict[str, Any]]) -> Tuple[int, int, int]:
             if response.status_code != 200:
                 logging.error(f"Status code:{response.status_code}")
 
-            relation_scores = json.loads(response.content)
+            event_scores = json.loads(response.content)
```
Comment (ankushc48) [inline measurement_scripts/tag_measurement.py:92]: I am guessing this is out of confusion due to 2 PRs opened at the same time. 🤣 This is still `tag_measurement` script.
you can resolve it when you see the conflicts before merging, depending upon whichever PR you merge first.

**Review by ankushc48 — APPROVED** [review]
Looks good 👍. 
Just be aware merge conflicts while merging

File: measurement_scripts/event_measurement.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    """
+    true_positive, false_negative, false_positive = 0, 0, 0
+    for true_event in true_events:
+        for pred_event in pred_events:
+            if "compound" not in true_event or "compound" not in pred_event:
+                true_event["compound"] = ""
+                pred_event["compound"] = ""
+            if (
+                true_event["disease"] == pred_event["disease"]
+                and true_event["date"].split("T")[0] == pred_event["date"].split("T")[0]
```
Comment (ceru23) [inline measurement_scripts/event_measurement.py:44]: I hate milliseconds too

File: measurement_scripts/event_measurement.py [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+def create_true_pred_list(
+    true_events: List[Dict[str, Any]],
+    pred_events: List[Dict[str, Any]],
+) -> Tuple[int, int, int]:
+    """
+    Creates list of true and prediction labels with 1s and 0s.
+
+    Args:
+        true_tags (List[Dict[str, Any]]): True tags, ground truth values
```
Comment (ceru23) [inline measurement_scripts/event_measurement.py:30]: I think that you left the params description of the ner function

**Review by ceru23 — APPROVED** [review]
LGTM, I agree with the comments by Ankush, and just a minor in the func description

**Pull request merged**